website/docs/infrastructure/troubleshooting/appservice-hidden-appsettings.md

```diff
@@ -0,0 +1,139 @@
+---
+sidebar_label: Hidden AppService/FunctionApp AppSettings in Terraform Plan
```

Comment:
Try to put the expected outcome in the title. For example this is a good title "Ensuring an Online Service is Reachable from an App Service Plan"

Reply:
https://github.com/pagopa/dx/pull/338/commits/8a08452798704902a9c0730d78e98cb4cb6c50e7